proto/storage/config.proto

```diff
@@ -55,8 +55,11 @@ message DecommissionedClusterRetentionConfig {
     google.protobuf.Timestamp created_at       = 4;
 }
 
+// next available tag: 4
 message ReportRetentionConfig {
-    int32 history_retention_duration_days       = 1;
+    int32 history_retention_duration_days              = 1;
+    uint32 downloadable_report_retention_days          = 2;
```

Comment:
It looks like uint32 should be better. We could make a dedicated PR to change if needed.

Reply:
That makes sense since the number of days is >= 0